File: programmers/lv2/n진수게임.js
function solution(n, t, m, p) {
  let [converted, result] = ["", ""];
  for (let i = 0; i < t * m; i += 1) converted += i.toString(n);
  for (let i = p - 1; i < t * m; i += m) result += converted[i];
  return result.toUpperCase();
}

File: programmers/lv2/n진수게임.py
def solution(n, t, m, p):
    import string
    tmp = string.digits + string.ascii_uppercase
    def convert(num, base) :
        q, r = divmod(num, base)
        if q == 0 :
            return tmp[r] 
        else :
            return convert(q, base) + tmp[r]
    converted = ""
    for i in range(t * m):
        converted += convert(i, n)
    return "".join([converted[i] for i in range(p - 1, t * m, m)])
    